lib/src/base-styles/_nx-btn.scss

```diff
@@ -11,107 +11,110 @@
   @include semi-bold();
   @include container-horizontal;
 
-  background-color: $nx-grey-1;
-  border: $nx-border-dark;
-  border-radius: 4px;
-  color: $nx-text-color-light;
+  background-color: $nx-white;
+  border: 1px solid #045dd1;
+  border-radius: 6px;
+  color: #045dd1;
   cursor: pointer;
   display: inline-block;
-  font-size: 14px;
-  height: 32px;
+  font-size: 16px;
+  height: 40px;
   line-height: 16px;
-  margin: 0 8px;
-  padding: 0 12px;
+  margin: 0 $nx-spacing-xs;
+  padding: 0 $nx-spacing-l;
   text-align: center;
   vertical-align: middle;
   min-width: 100px;
 
-  + .nx-btn {
-    // buttons should be 8px from adjacent items, except other buttons which they should only be 4px from
-    margin-left: -4px;
+  &:hover {
+    background-color: #c5deff;
   }
 
-  &:hover, &:focus, &:active {
-    border-color: $nx-slate;
-    color: $nx-text-color;
+  &:focus {
+    border: 1px solid #43cbf5;
```

Comment:
FYI this border inevitably appears on click as well, since the click focuses the button.  Makes sense to me but strictly speaking it's not what's showing in the mockup.